src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperCloneCheck.java

```diff
@@ -37,7 +37,45 @@
  * <pre>
  * &lt;module name=&quot;SuperClone&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Foo {
+ *  protected int f;
+ *
+ *  public Foo() {};
+ *  public Foo(int f) {
+ *     this.f = f;
+ *  }
+ *  public Object clone() { // violation, does not call super.clone()
+ *    Foo other = new Foo();
+ *    other.f = this.f;
+ *    return other;
+ *  }
+ * }
+ *
+ * class FooA extends Foo {
+ *  private static int countA = 0;
+ *  private int a;
  *
+ *  public Object clone() { // OK
+ *   FooA other = (FooA) super.clone();
+ *   other.a = ++countA;
+ *   return other;
+ *  }
+ * }
+ *
+ * class FooB extends Foo {
+ *  private static int countB = 0;
+ *  private int b;
+ *
+ *  public Object clone() { // violation, does not call super.clone()
```

Comment:
Please show example of clone method that return exact class ( FooB ).
Please read https://github.com/checkstyle/checkstyle/issues/7553#issuecomment-590786496 and articles we reference.